tests/bluetooth/audio/ascs/src/test_ase_state_transition.c

```diff
@@ -93,10 +114,29 @@ ZTEST_F(test_sink_ase_state_transition, test_client_codec_configured_to_qos_conf
 
 	/* Preamble */
 	test_ase_control_client_config_codec(conn, ase_id, stream);
+	test_mocks_reset();
 
 	test_ase_control_client_config_qos(conn, ase_id);
+
+	/* Verification */
 	expect_bt_bap_unicast_server_cb_qos_called_once(stream, EMPTY);
+	expect_bt_bap_unicast_server_cb_config_not_called();
+	expect_bt_bap_unicast_server_cb_reconfig_not_called();
+	expect_bt_bap_unicast_server_cb_enable_not_called();
+	expect_bt_bap_unicast_server_cb_start_not_called();
+	expect_bt_bap_unicast_server_cb_metadata_not_called();
+	expect_bt_bap_unicast_server_cb_disable_not_called();
+	expect_bt_bap_unicast_server_cb_stop_not_called();
+	expect_bt_bap_unicast_server_cb_release_not_called();
```

Comment:
Hmm, it seems overkill to define which functions are not called, instead of just the ones that are called. 

Could we not define these test so by default it expects none of them called, and then only define which should be called here?

Reply:
Sounds reasonable